src/hotspot/share/jfr/support/jfrNativeLibraryLoadEvent.cpp

```diff
@@ -32,7 +32,9 @@
 JfrNativeLibraryEventBase::JfrNativeLibraryEventBase(const char* name) : _name(name), _error_msg(nullptr), _start_time(nullptr) {}
 
 JfrNativeLibraryEventBase::~JfrNativeLibraryEventBase() {
-  delete _start_time;
+  if (_start_time != nullptr) {
+    delete _start_time;
+  }
```

Comment:
This should not be necessary. Delete should not fail on `nullptr`.

Reply:
start_time is a JfrCHeapObj, an allocate that hook allocations and deallocations. With a direct delete it will account the size of the object even though its null. Should probably be fixed in the allocator, but I needed this to get the accounting back on track.

Reply:
Thanks for the explanation. I got a bit confused ...